app/Http/Controllers/PagesController.php

```diff
@@ -48,6 +51,12 @@ public function login(Request $request)
             $username = $request->input('username');
             $password = $request->input('password');
 
+            $user = User::where('username', $username)->first();
+
+            if ($user->confirmed == false) {
+                return \Redirect::back()->withErrors(trans('messages.notConfirmed'));
+            }
+
```

Comment:
Please move this section to below the Auth::attempt if statement. If a user submits an empty username they get an error generated from User::where.